### PR TITLE
Include build number in generated files

### PR DESCRIPTION
**Pull request by superyu1337**
PR to address #45, which I have opened some days ago.
This change creates a new container in the offsets file called `game_info`.  
For now this container gets created when `dwBuildNumber` is being read, and is being populated with one entry.  
Said entry is the build number, at the time of the dumping process.  
If there will be any other "dump-time" variables added to the container in the future, the relevant code should probably be relocated.

Long story short, in the end it basically adds this to the offset files (Here in rust):
```rust
pub mod game_info { // Some additional information about the game at dump time
    pub const buildNumber: usize = 0x369F; // Game build number
}
```

### CODE REVIEW

**Comment by a2x** [comment]
Thanks for the PR. I'll merge this for now.

I think that separating this from the generated offset files would be better, however. One of the reasons is that depending on the file type, the value will be displayed in hexadecimal, which may not be ideal if you manually check the file for the build number.
Initially, I thought about postfixing each generated file name with the build number, but this would make parsing it difficult, which isn't ideal. Another approach is to generate a single text file called `build.txt` or similar, containing just the build number; it would be easy to parse and check manually. If there was a need to store other info collected at dump time like you mentioned, then a single JSON file may be better. I will reconsider this when I have some time again.

What are your thoughts on this?

**Comment by superyu1337** [comment]
I'm not sure about a singular build.txt file, I mean, it will work, but I personally like having the stuff compiled in, so no extra parsing is necessary.
My idea would be to create a new file`, and using different filebuilders, or modifying the existing ones to have a toggle for hexadecimal values. This would be a futureproof solution, for when new dump-time values get added. But to be honest, I don't know what else could be added?
I think I'll leave the decision up to you, it's your project after all 😆
